### PR TITLE
feat(tools/readme_linter): Check for global configuration section

### PR DESCRIPTION
**Pull request by Hipska**
- Let the CI also enforce the new README section introduced by #12107.
- Update the include section with a more correct link and update this to the `example` input.
- Add/update this section also to all plugins.

# Required for all PRs

- [x] Updated associated README.md.
- [ ] Wrote appropriate unit tests.
- [x] Pull request title or commits are in [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0/#summary)

### CODE REVIEW

File: docs/includes/plugin_config.md [hunk 1]
```diff
@@ -3,4 +3,4 @@ additional global and plugin configuration settings. These settings are used to
 modify metrics, tags, and field or create aliases and configure ordering, etc.
 See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
 
-[CONFIGURATION.md]: ../../../docs/CONFIGURATION.md
+[CONFIGURATION.md]: ../../../docs/CONFIGURATION.md#plugins
```
Comment (powersj) [inline docs/includes/plugin_config.md:6]: Making this change will modify all plug readmes, so you will need to run `make docs` along with this PR.

Comment (Hipska) [reply docs/includes/plugin_config.md:6]: I asked this on Slack, but didn’t get any response yet. I did it already but hadn’t pushed yet.

**Comment by Hipska** [comment]
@powersj did pull master and make docs and all seems to be clean. I don't get why the CI thinks it's not. 🤷

**Comment by powersj** [comment]
> @powersj did pull master and make docs and all seems to be clean. I don't get why the CI thinks it's not. shrug

heh so two things:

1) Your master is behind. It doesn't include some of the new plugins.
2) I assume you are doing this on a mac? I think the build flags are preventing you from updating the linux only plugins as the diff for me shows that is what makes up most of the missing changes:

```
diff --git a/plugins/inputs/conntrack/README.md b/plugins/inputs/conntrack/README.md
diff --git a/plugins/inputs/dpdk/README.md b/plugins/inputs/dpdk/README.md
diff --git a/plugins/inputs/hugepages/README.md b/plugins/inputs/hugepages/README.md
diff --git a/plugins/inputs/intel_dlb/README.md b/plugins/inputs/intel_dlb/README.md
diff --git a/plugins/inputs/intel_pmu/README.md b/plugins/inputs/intel_pmu/README.md
diff --git a/plugins/inputs/intel_powerstat/README.md b/plugins/inputs/intel_powerstat/README.md
diff --git a/plugins/inputs/iptables/README.md b/plugins/inputs/iptables/README.md
diff --git a/plugins/inputs/ipvs/README.md b/plugins/inputs/ipvs/README.md
diff --git a/plugins/inputs/kernel/README.md b/plugins/inputs/kernel/README.md
diff --git a/plugins/inputs/kernel_vmstat/README.md b/plugins/inputs/kernel_vmstat/README.md
diff --git a/plugins/inputs/linux_cpu/README.md b/plugins/inputs/linux_cpu/README.md
diff --git a/plugins/inputs/mdstat/README.md b/plugins/inputs/mdstat/README.md
diff --git a/plugins/inputs/ras/README.md b/plugins/inputs/ras/README.md
diff --git a/plugins/inputs/sensors/README.md b/plugins/inputs/sensors/README.md
diff --git a/plugins/inputs/slab/README.md b/plugins/inputs/slab/README.md
diff --git a/plugins/inputs/sysstat/README.md b/plugins/inputs/sysstat/README.md
```

**Comment by Hipska** [comment]
I did merge this (my morning) with 6fb08bb3d which was the latest at that time. Strange it wouldn't update all plugin README's, those are not supposed to be platform dependant?

**Comment by Hipska** [comment]
@powersj I added the updated docs for other GOOSs

**Review by powersj — APPROVED** [review]

**Comment by powersj** [comment]
Thanks!

**Review by srebhan — APPROVED** [review]
Good job @Hipska! Looks good to me.

**Pull request merged**